tests/python/quantization/test_quantization.py

```diff
@@ -319,6 +339,11 @@ def check_quantized_flatten(shape):
 
 @with_seed()
 def test_quantize_params():
+    if is_test_for_mkldnn():
+        dtype = 'uint8'
```

Comment:
Why different dtypes? the test is written to target int8, so why test uint8 now?

Reply:
Currently MKLDNN quantization only support uint8 as input data so we made separate logic for mkldnn.